bootstrap.py

```diff
@@ -122,47 +112,59 @@ def configureLogging():
 """)
 
 
-def makeInputs(tracts, filters):
-    instrument = HyperSuprimeCam()
-    raws = []
+def makeVisitList(tracts: List[int], filters: List[str]):
+    visits = []
     for tract in tracts:
         for filter in filters:
-            raws.extend(computeFilesForVisits(VISITS[tract][filter]))
-    return BootstrapRepoInputs(instrument=instrument, raws=raws,
-                               refCatRoot=REFCAT_ROOT,
-                               brightObjectMaskRoot=BRIGHT_OBJECT_MASK_ROOT,
-                               calibRoot=GEN2_CALIB_ROOT)
+            visits.extend(VISITS[tract][filter])
+    return visits
 
 
-def makeTask(butler):
+def makeTask(butler: Butler, *, continue_: bool = False):
     instrument = HyperSuprimeCam()
-    config = BootstrapRepoTask.ConfigClass()
-    instrument.applyConfigOverrides(BootstrapRepoTask._DefaultName, config)
-    config.raws.onError = "break"
-    config.raws.transfer = "symlink"
-    for sub in config.refCats.values():
-        sub.transfer = "symlink"
-    config.brightObjectMasks.transfer = "symlink"
-    config.calibrations.transfer = "symlink"
-    return BootstrapRepoTask(config=config, butler=butler)
-
-
-def main(root, *, tracts, filters, create=False, clobber=False):
+    config = ConvertRepoTask.ConfigClass()
+    instrument.applyConfigOverrides(ConvertRepoTask._DefaultName, config)
+    config.relatedOnly = True
+    config.transfer = "symlink"
+    config.rootSkyMapName = "hsc_rings_v1"
```

Comment:
Why not add a default of `rootSkyMapName` to `obs_subaru/config/hsc/convertRepo.py` as "hsc_rings_v1" already looks like the de facto default there? 

Reply:
Will do.  At the time, I was thinking of how ci_hsc uses a different (discrete) SkyMap, but since essentially every other HSC repo does use `hsc_rings_v1` it does make sense to make this an instrument-wide default.